pxr/usd/usdShade/materialBindingAPI.cpp

```diff
@@ -252,7 +252,7 @@ UsdShadeMaterialBindingAPI::DirectBinding::DirectBinding(
 UsdShadeMaterial
 UsdShadeMaterialBindingAPI::DirectBinding::GetMaterial() const
 {
-    if (!_materialPath.IsEmpty()) {
+    if (_bindingRel && !_materialPath.IsEmpty()) {
```

Comment:
Thank you, @tkchanat !  Can you change this pattern to `if (_bidingRel.GetPrim() ... ` ?  It is sufficient, and cheaper than a full Relationship validity test.  Thank you!

Reply:
Thanks for the review @spiffmon! I changed the pattern in the latest commit and it's looking good now.